1.13_BouncingBalls/src/bouncingBallsPackage/Ball.java

```diff
@@ -78,12 +77,8 @@ public void reflectVertical(){
 	}
 	
 	public String toString() {
-		String chain;
-		chain = "Ball at ("+x+","+ y+") of velocity ("+xDelta+","+ yDelta+")";
-		return chain;
+		return ("Ball at ("+x+","+ y+") of velocity ("+xDelta+","+ yDelta+")");
```

Comment:
Why parenthesis ? Does this work ? It should be return " ..." without parenthesis.